completions/ssh.completion.sh

```diff
@@ -45,7 +45,11 @@ function _omb_completion_ssh {
     fi
   done
   if ((${#config_files[@]} != 0)); then
-    COMPREPLY+=($(compgen -W "$(awk '/^Host/ {for (i=2; i<=NF; i++) print $i}' "${config_files[@]}")" "${options[@]}"))
+    COMPREPLY+=($(compgen -W "$(awk '
+      sub(/^[ \t]*[Hh][Oo][Ss][Tt]([Nn][Aa][Mm][Ee])?[ \t=]+/, "") {
+        n = split($0, fields, /[ \t]+/);
+        for (i = 1; i <= n; i++) if (fields[i] && !visited[fields[i]]++) print fields[i];
```

Comment:
Hmm, if we just use `fields[i]` as a condition, the hostname shortcut `0` would be excluded. We should explicitly check that the field is a non-empty string.

```suggestion
        for (i = 1; i <= n; i++) if (fields[i] != "" && !visited[fields[i]]++) print fields[i];
```